tests/testthat/test-xml_name.R

```diff
@@ -1,3 +1,13 @@
+test_that("xml_name() returns the name", {
+  x <- sample_nodeset()
```

Comment:
IMO it would be better to copy and paste the contents of `sample_nodeset()` here; otherwise it's hard to verify that the test is correct.